flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/RestHandlerConfiguration.java

```diff
@@ -46,6 +51,8 @@ public class RestHandlerConfiguration {
     public RestHandlerConfiguration(
             long refreshInterval,
             int maxCheckpointStatisticCacheEntries,
+            Duration checkpointStatsSnapshotCacheExpireAfterWrite,
+            int checkpointStatsSnapshotCacheSize,
```

Comment:
It is confusing to understand the difference between `maxCheckpointStatisticCacheEntries` and `checkpointStatsSnapshotCacheSize` without reading the code. Can the variable names be improved?

Reply:
Ok renamed.